aws-py-voting-app/blogpost.md

```diff
@@ -1,12 +1,13 @@
-    Creating a Python AWS Application Using Flask, Redis, and Pulumi
+# Creating a Python AWS Application Using Flask, Redis, and Pulumi
 
-Having recently begun developing with Pulumi, I have taken it upon myself to learn as much about its inner workings and processes as I could. To that end, I have decided to construct a production-level application using it, and to document each step that I take and its impact on my progress as I go along. 
+I've recently begun developing with Pulumi to learn as much about its inner workings and processes as I could. I decided to construct a production-level application and to document each step that I take and my progress as I go along.
```

Comment:
Suggested introduction:

Meet Vova Ivanov - one of the Pulumi summer interns. He'll be writing about his experiences learning Pulumi while modernizing a web app and its underlying infrastructure.